stacks/upload-db-stack.js

```diff
@@ -23,8 +29,14 @@ export function UploadDbStack({ stack, app }) {
    */
    const uploadTable = new Table(stack, 'upload', uploadTableProps)
 
+   /**
+    * This table tracks count of uploads per space.
+    */
+   const spaceUploadCountTable = new Table(stack, 'space-upload-count', spaceUploadCountTableProps)
```

Comment:
Do you think we should have a `upload-metrics` table instead rather than having a table per metric we want to collect?

"upload" because the metrics relate to the `upload/*` invocations.

So then you'd have a key as "space DID + metric name" and you can fetch all metrics for a given space with one query.

Reply:
...and then the metric name might be `add-count` - as it's the count of the number of `upload/add` invocations.

As with the other PR, we should collect `remove-count` also.

Reply:
...although thinking about it, why separate it, why not just have a generic "metrics by space" table with key as "space DID + metric name" and metric names as `upload-add-count` and `upload-remove-count`.

Reply:
@alanshaw I think your suggestion makes sense!

At this point, considering if that should leave in a `metrics-db-stack` then, instead of within upload stack. What do you think?